lsp-csharp.el

```diff
@@ -234,19 +234,20 @@ tarball or a zip file (based on a current platform) to TARGET-DIR."
 
    (t (error "lsp-csharp cannot extract \"%s\" on platform %s (yet)" filename system-type))))
 
+(defun lsp-csharp--language-server-path ()
+  "Resolves path to use to start the server."
+  (if lsp-csharp-server-path lsp-csharp-server-path
```

Comment:
nit: use `when` instead of `if` when there is only `then` clause

Reply:
Ok sounds good, I'll force push a patch.

Reply:
Oh wait, sorry both branches are being used so a `when` won't work here. Do you want me to move the true branch to the next line so it's clearer?

Reply:
Oh, yeah, having either all on one line or on 3 lines.